packages/design-system/src/types/Table/Table.d.ts

```diff
@@ -30,8 +20,23 @@ export interface TableProps {
    * This prop will only be used when the `Table` `scrollable` prop is set and the table width is wider than the viewport.
    */
   scrollableNotice?: React.ReactNode;
+  /**
+   * A stackable variation of the table.
+   * When `stackable` is set, `id` or `headers` prop is required in `Table`
+   */
+  stackable?: boolean;
+  /**
+   * Applies responsive styles to vertically stacked rows at different viewpoint sizes.
+   */
+  stackableBreakpoint?: TableStackableBreakpoint;
```

Comment:
This is something we can iterate on, but it might be nice later on to add a `stackableHeaderWidth` prop or something, for adjusting the percentage in case 40% doesnt work

Reply:
I've changed it to max-width of 120px basing off the uswds template and it would be good to add `stackableHeaderWidth` prop for adjusting the header size. Will note down.